core/misc/jquery.once.js

```diff
@@ -1,10 +1,8 @@
 /**
- * jQuery Once Plugin 1.2.6
- * http://github.com/robloach/jquery-once
+ * @file
+ * jQuery Once Plugin
  *
- * Dual licensed under the MIT and GPL licenses:
- *   http://www.opensource.org/licenses/mit-license.php
```

Comment:
These lines should be kept, and the new line added. See the [system.tar.inc](https://github.com/backdrop/backdrop/blob/1.x/core/modules/system/system.tar.inc) file where the copyright and the license information has not been removed.

Reply:
Possibly, but the situation seems a bit different here.

There won't be any more updates on the original jquery.once, so there's nothing to "merge" in the future.

I really hesitate to keep the (different) license, when we actually make this part of B, which is GPLv2.

Reply:
GPLv2 requires to keep the original copyright notice. It also requires to state any significant changes made to the original software, like other licenses do.

That is required even if I take a project that is not updated from years and I start to change its files and publish them in a different repository. It is not a matter of updating my code with recent changes done from the code I copied.

Reply:
I'm no lawyer, so have to rely on your statements.